Octokit/Models/Response/User.cs

```diff
@@ -23,6 +27,13 @@ public User(string avatarUrl, string bio, string blog, int collaborators, string
         /// </summary>
         public bool SiteAdmin { get; protected set; }
 
+        /// <summary>
+        /// LDAP Binding (GitHub Enterprise only)
+        /// </summary>
+        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Dn")]
+        [SuppressMessage("Microsoft.Naming", "CA1709:IdentifiersShouldBeCasedCorrectly", MessageId = "Dn")]
+        public string LdapDn { get; protected set; }
```

Comment:
Alternatively, I think this could be added to the `CustomDictionary.xml` at the root of the repository.